### PR TITLE
Updated image details to point people to Quay

### PR DESCRIPTION
**Pull request by mattfarina**

### CODE REVIEW

**Comment by rimusz** [comment]
LGTM

File: build.sh [hunk 1]
```diff
@@ -19,6 +19,10 @@ set -o nounset
 set -o pipefail
 
 readonly IMAGE_TAG=v1.0.5
+
+# The image goes into two repoistories. quay.io/helmpack/chart-testing is used
```
Comment (unguiculus) [inline build.sh]: repoistories -> repositories

Comment (mattfarina) [reply build.sh]: I just learned I didn't have spell check turned on for shell files. Thanks.

File: README.md [hunk 2]
```diff
@@ -143,7 +143,7 @@ Done.
 ### Installing and Testing Charts
 
 Installing a chart requires access to a Kubernetes cluster.
-You may have to create your own Docker image that extends from `gcr.io/kubernetes-charts-ci/chart-testing:v1.0.5` in order to install additional tools (e. g. `google-cloud-sdk` for GKE).
+You may have to create your own Docker image that extends from `quay.io/helmpack/chart-testing:v1.0.5` in order to install additional tools (e. g. `google-cloud-sdk` for GKE).
 You could run a container in the background, run the required steps to authenticatre and initialize the `kubectl` context before you, and eventually run `chart_test.sh`.
```
Comment (unguiculus) [inline README.md]: While you're at it. Could you fix this sentence, please?

**Pull request merged**